src/MilošVojnović/Nasledjivanje.java

```diff
@@ -5,7 +5,7 @@ interface Auto {
         public void operationA();
         public void operationB();
     }
-
+//promena
     interface Biciklo {
```

Comment:
**Use English as a main language for programming.**
Rename each name that you use in the code from Serbian to English.
If you do not know English very good, **please use Google Translate to translate words - it is very important.**
**In professional programming you will always use English.**